docs/sdk/analytics/fragments/ios/getting-started.md

```diff
@@ -139,11 +141,11 @@ Build and run your app to see usage metrics in Amazon Pinpoint. When you run the
 1. To see visualizations of the analytics coming from your app, open your project in the Amazon Pinpoint console by running the following:
 
 ```bash
-$ amplify console analytics
+$ amplify analytics console
```

Comment:
I'm pretty sure its `amplify console analytics` @kaustavghosh06

Reply:
This just worked for me, and I think makes more phonetic sense which is why we originally aliased the commands

Reply:
Don't disagree or agree.. we're just using `amplify console <category>` in a bunch of the doc pages already.